x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/connectors/connectors.spec.ts

```diff
@@ -47,7 +47,10 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
         endpoint: 'GET /internal/observability_ai_assistant/connectors',
       });
 
-      expect(res.body.length).to.be(1);
+      const connectorsExcludingPreconfiguredInference = res.body.filter(
+        (c) => c.actionTypeId !== '.inference'
+      );
+      expect(connectorsExcludingPreconfiguredInference.length).to.be(1);
```

Comment:
This lgtm - although is there any reason to filter it out instead of including preconfigured connectors in the count?

```suggestion
      expect(connectorsExcludingPreconfiguredInference.length).to.be(2);
```

Reply:
I'm wasn't sure whether this will show up in stateful (or when it'll show up in stateful)
By filtering it out, even if it doesn't show up in stateful, the test will pass.